frontend/src/store/objectStore.ts

```diff
@@ -62,7 +62,7 @@ export const useObjectStore = defineStore('objectStore', () => {
         // TODO: Tags
 
         // Populate object
-        if (metadataResponse.data.length && metadataResponse.data[0]) {
+        if (metadataResponse.data[0] && metadataResponse.data.length) {
```

Comment:
Not completely sure, but might be able to get away with just one conditional check as the second conditional is redundant?
```ts
        if (metadataResponse.data[0]) {
```